backend/codesirius/codesirius/views.py

```
@@ -1,7 +1,17 @@
+from rest_framework.decorators import api_view
 from rest_framework.exceptions import NotFound
+from rest_framework.response import Response
 from rest_framework.views import APIView
 
 
+@api_view(["GET"])
+# Restricted to GET requests only.
+# Health check endpoint.
+# Safe because it only returns a status and has no side effects.
+def health_check(request):
```

Ruff (0.8.2)

Missing return type annotation for public function `health_check`

(ANN201)